charts/apisix/values.yaml

```diff
@@ -186,6 +183,11 @@ admin:
     ipList:
       - 127.0.0.1/24
 
+nginx:
+  workerRlimitNofile: "1047552"
+  workerConnections: "16384"
```

Comment:
These two fields should be integer?

Reply:
nope, if those fields are number type, they will be converted into string type while helm command processing, for example, number 1047552 will be `1.047552e+06`, then cause apisix gateway fail to started.

Reply:
Gotcha. I think this is due to the handling of number in helm.